README.md

```diff
@@ -1,56 +1,24 @@
-
-### Features
-The approach to this emulator is totally different from other emulators. It focuses on a framework on which many computer systems can be easily emulated. The features that make this possible are:
-
-#### State-of-the-art User Interface
-The first pillar of OpenEmulator is a state-of-the-art user interface. Emulations consist of an emulation window that gives you an overview of the installed devices/free ports, and also shows the details and properties of a device. It is thus easy to change the settings of a video monitor, a device's DIP switches or to configure a joystick.
-
-The Hardware Library lets you easily install new devices: just pick a device from the Hardware Library list, and drag it over to an empty port in the emulation window. And all of this even works in real time!
-
-Devices like displays, printers or keypads usually have a separate window. You can display these by clicking on the magnifying buttons.
+libemulation provides a component framework and library to accurately emulate legacy computer systems.
 
-In order to mount disk or ROM images you simply drag them to a storage device in the emulation window. You can also drag them over to any other window of the emulation or the app icon to let OpenEmulator decide the best device for this operation. To unmount a disk or ROM image press the eject button next to it.
+Emulation is done by connecting emulated components together using an XML description called "Emulation Description Language". It is thus very easy to interconnect different CPUs, address decoders, memories, chip-sets or input/output devices to fully emulate different computer systems.
 
-#### Video Rendering
-A second pillar is awesome GPU-accelerated video rendering. There is support for accurate CRT and LCD emulations, paper printouts and native OpenGL output.
+It is also easy to add your own software components to the framework. You are encouraged to submit your own emulations as pull requests, so your work gets all the attention it deserves.
 
-The effects that are available for video rendering are: NTSC/PAL decoding, standard monitor controls like brightness, contrast, saturation, hue, active video center/size controls, monitor shadow masks like triad, in-line, aperture grille, LCD, Bayer, a CRT barrel effect, CRT dynamic scanline effect, CRT phosphor persistence, CRT center lighting, video bandwidth controls, composite color IRE levels, composite color luma and chroma bandwidth, composite color per-line variable color burst, composite color per-line U phase alternation, composite color Y'UV, Y'IQ (with standard I and Q bandwidth) and Sony decoding.
+## Usage
+libemulation is implemented in C++ and can be linked with your emulator application as a static library. We aim to be portable, but currently the only offically supported and tested platform is macOS. Refer to [INSTALL.md](INSTALL.md) for instructions how to build libemulation.
 
-#### Component Framework
```

Comment:
Out of curiosity, why remove this information?

Reply:
This is an effort to reduce duplicated information, the README.md file on the master now is an exact copy of the one in [OpenEmulator](https://github.com/openemulator/libemulation/blob/master/README.md). I have some upcoming side patches that will clean up the documentation a bit.

Reply:
lgtm